analytics/pubstack/config.go

```diff
@@ -2,24 +2,38 @@ package pubstack
 
 import (
 	"encoding/json"
+	"fmt"
+	"io"
 	"net/http"
 	"net/url"
 	"time"
 
 	"github.com/docker/go-units"
+	"github.com/golang/glog"
 )
 
 func fetchConfig(client *http.Client, endpoint *url.URL) (*Configuration, error) {
 	res, err := client.Get(endpoint.String())
```

Comment:
Do we want to add a timeout to this fetch while we are here?

Reply:
I agree we should. I don't know the pubstack analytics module well enough to implement it. Will reach out to pubstack to implement.